netgrasp/netgrasp.py

```diff
@@ -33,7 +35,6 @@
 PROCESSED_WEEKLY_DIGEST = 4
 PROCESSED_NOTIFICATION  = 8
 
-DEFAULT_CONFIG    = ['/etc/netgrasp.cfg', '/usr/local/etc/netgrasp.cfg', '~/.netgrasp.cfg', './netgrasp.cfg']
```

Comment:
I see now that this was actually moved, not duplicated -- GitHub hid the changes too this file as there were too many (not a good sign ;)